### PR TITLE
Create using_development_nuget_packages.md

### PR DESCRIPTION
**Pull request by dellis1972**
Add some documentation on how to use the GitHub NuGet Feed.

### CODE REVIEW

**Comment by dellis1972** [comment]
Don't merge this year. I want to double check it. I just flagged it for a review.

**Comment by dellis1972** [comment]
ok , should be good to go now

File: articles/getting_started/using_development_nuget_packages.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
+# Using the Development Nuget Packages
```
Comment (SimonDarksideJ) [inline articles/getting_started/using_development_nuget_packages.md:1]: Header needs the front matter used by the doc generator and remove the h1 element (it should be the title)

File: articles/getting_started/using_development_nuget_packages.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+            <add key="Username" value="%GITHUB_USER%" />
+            <add key="ClearTextPassword" value="%GITHUB_TOKEN%" />
+        </MonoGameGitHub>
+    </packageSourceCredentials>
+</configuration>
+```
+
+The really good thing about placing these credentials outside of source control is that they are safe. But also any
+new projects you create under that folder can also make use of these createntials. So it is a good idea to keep them in one place.
+For more information, you can read [consuming-packages-authenticated-feeds](https://learn.microsoft.com/en-us/nuget/consume-packages/consuming-packages-authenticated-feeds#credentials-in-nugetconfig-files).
```
Comment (SimonDarksideJ) [inline articles/getting_started/using_development_nuget_packages.md:131]: Needs a trailing space for markdown spec

File: articles/getting_started/using_development_nuget_packages.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
+# Using the Development Nuget Packages
+
+When the MonoGame develop branch builds, it publishes development NuGet packages to the 
+MonoGame NuGet Feed on GitHub. If you want to test a new feature or just be on the 
+very latest code you can use this Feed to do that.
```
Comment (SimonDarksideJ) [inline articles/getting_started/using_development_nuget_packages.md:5]: Can we also add a ;tldr breakdown list of steps to do

**Review by SimonDarksideJ — CHANGES_REQUESTED** [review]

**Review by SimonDarksideJ — APPROVED** [review]

**Pull request merged**